idflakies-maven-plugin/src/main/java/edu/illinois/cs/dt/tools/plugin/DetectorMojo.java

```diff
@@ -195,20 +194,31 @@ public int moduleRounds(String coordinates) throws IOException {
 
     @Override
     public void execute() {
-        super.execute();
+        superExecute();
         this.outputPath = PathManager.detectionResults();
 
         final ErrorLogger logger = new ErrorLogger();
         this.coordinates = mavenProject.getGroupId() + ":" + mavenProject.getArtifactId() + ":" + mavenProject.getVersion();
 
+        long startTime = System.currentTimeMillis();
         logger.runAndLogError(() -> detectorExecute(logger, mavenProject, moduleRounds(coordinates)));
+        timing(startTime);
     }
 
-    private Void detectorExecute(final ErrorLogger logger, final MavenProject mavenProject, final int rounds) throws IOException {
+    protected void superExecute() {
+        super.execute();
+    }
+
+    protected Void defineSettings(final ErrorLogger logger, final MavenProject project) throws IOException {
```

Comment:
To allow for somewhat easier adjustments now that we are planning to have subclasses of `DetectorMojo`, let's not have `detectorExecute` call `defineSettings` but instead have it explicitly called in `execute`. We could move the call to `loadTestRunners` into the beginning of `detectorExecute` (since mainly `detectorExecute` needs the test runners).

Reply:
Also should delete the time file as to ensure subsequent runs do not keep appending new times into the same file.